content/apps/oauth2-scopes.md

```diff
@@ -0,0 +1,52 @@
+# OAuth scopes
+
+During App authorization, App must specify the minimal set of scopes that is required for the App to function.
+
+Scopes limits App access to the **PIM REST API**. Once App is authorized, it can only interact with API when in compliance with scopes requested.
+
+::: warning
+Scopes cannot be changed once activation process is over.
+If your App needs to update the list of required scopes, it must redo the process of activation.
```

Comment:
This is confusing and should be clarified because an App cannot redo by itself the process of activation.